packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -684,11 +625,19 @@ contract VotingV2 is
         return voteTiming.computeCurrentPhase(getCurrentTime());
     }
 
+    /**
+     * @notice Returns the phase length set in vote timing.
+     * @return uint256 the phase length.
+     */
+    function getPhaseLength() public view returns (uint256) {
+        return voteTiming.phaseLength;
+    }
+
```

Comment:
As we discussed in the other PR, is this method necessary if we have voteTiming as a public variable? Removing this might also save some bytecode.